src/eventProcessor.ts

```diff
@@ -21,10 +21,25 @@ import {
   handleEditColonyAction,
   handleVersionUpgradeAction,
   handleMotionCreated,
+  handleMotionStaked,
 } from './handlers';
+import { RemoveListener } from '~utils';
 
 dotenv.config();
 
+export enum RemoveListenerKeys {
+  EXTENSION_INITIALISED = 'extensionInitialised',
+}
+
+export interface EventProcessorContext {
```

Comment:
Allows the listeners to be removed by the `extensionUninstalled` handler that were generated in `extensionInitialised`